wallets/react-wallet-v2/yarn.lock

```diff
@@ -1903,7 +1903,7 @@
     "@stablelib/constant-time" "^1.0.1"
     "@stablelib/wipe" "^1.0.1"
 
-"@stablelib/sha256@1.0.1":
+"@stablelib/sha256@1.0.1", "@stablelib/sha256@^1.0.1":
```

Comment:
Looks like 2 similarish versions of same lib here, maybe lockfile needs to be deleted and re-generated?

Reply:
I tried it but it's regenerated the same way. May be inner dependencies have it declared differently